harz_0_1_1.py

```diff
@@ -64,7 +61,8 @@ def runWochenende(stage_infile):
     # run Wochenende pipeline
     global dirNameWochenende
     stage = "run Wochenende pipeline"
-    dirNameWochenende = "analysis/" + getDate() + "Wochenende" + wochenendeVersion
+    #edit analysis path as needed
+    dirNameWochenende = "/path/to/analysis/" + getDate() + "Wochenende" + wochenendeVersion
```

Comment:
I think you can leave the path/to/analysis as analysis so the tool is ready to go out of the box. You'll be the main user, we're not intending to spread this widely I guess. Even if so, a subdirectory analysis in the current dir is no problem